tests/Main.hs

```diff
@@ -599,9 +599,10 @@ runClashTest = defaultMain $ clashTestRoot
             , buildTargets=BuildSpecific["top_bit", "top_bitvector", "top_index", "top_signed", "top_unsigned"]
             }
         , runTest "T2046B" def{clashFlags=["-Werror"]}
-        , runTest "T2046C" def{hdlSim=[],clashFlags=["-Werror"],buildTargets=BuildSpecific["topEntity"]}
-        , runTest "T2097" def{hdlSim=[]}
-        , runTest "T2154" def{hdlTargets=[VHDL], hdlSim=[]}
+        , runTest "T2046C" def{hdlSim=False,clashFlags=["-Werror"],buildTargets=BuildSpecific["topEntity"]}
+        , runTest "T2097" def{hdlSim=False}
+        , runTest "T2154" def{hdlTargets=[VHDL], hdlSim=False}
+        , runTest "T2272" def{hdlTargets=[VHDL], hdlSim=False}
```

Comment:
`hdlSim` is a list of simulators to run since PR #2257. So the existing lines were correct and the new one needs `hdlSim=[]` as well.

Reply:
Yeah, seems like I messed up my rebase.